scripts/generate_schemas.py

```diff
@@ -2,7 +2,7 @@
 """Generate JSON schemas from the frameworks questions content.
 
 Usage:
-    generate-schemas.py --output-path=<output_path>
+    generate_schemas.py --output-path=<output_path>
```

Comment:
What's the purpose of the rename? `-` is what we seem to have settled on in `digitalmarketplace-scripts` repo.